<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_pay_widget_service_manual</class>
 <widget class="QDialog" name="w_scr_dish_pay_widget_service_manual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>frame_detail_show</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
      <item row="0" column="2" rowspan="7">
       <widget class="lds_key3x4" name="widget" native="true"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>付款</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="lds_doublespinbox" name="doubleSpinBox_num_cost"/>
      </item>
      <item row="2" column="1">
       <widget class="lds_doublespinbox" name="doubleSpinBox_num_service"/>
      </item>
      <item row="1" column="1">
       <widget class="lds_doublespinbox" name="doubleSpinBox_num_lowcost"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>金额</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="lds_doublespinbox" name="doubleSpinBox_num_realreceive"/>
      </item>
      <item row="6" column="1">
       <widget class="lds_doublespinbox" name="doubleSpinBox_num_money"/>
      </item>
      <item row="5" column="1">
       <widget class="lds_doublespinbox" name="doubleSpinBox_num_paid"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>应收</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>低消</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>已付</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>服务费</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="lds_doublespinbox" name="doubleSpinBox_num_blotout"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>抹零</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_doublespinbox</class>
   <extends>QDoubleSpinBox</extends>
   <header>lds_doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_key3x4</class>
   <extends>QWidget</extends>
   <header>lds_key3x4.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
